assembl/static2/js/app/pages/debate.jsx

```diff
@@ -30,11 +34,41 @@ class Debate extends React.Component {
   displayThumbnails() {
     this.setState({ isThumbnailsHidden: !this.state.isThumbnailsHidden });
   }
+  redirectToV1Threads() {
```

Comment:
I do also think it would fit better in another file, but I could not find any file more suited for this, and have not enough knowledge yet about the whole architecture of the React frontend to come up with a more suited one. I'll trust your judgement about the best place :)